modules/__tests__/Media-test.js

```diff
@@ -133,7 +133,7 @@ describe('A <Media>', () => {
         )
 
         render(element, node, () => {
-          expect(node.firstChild.innerHTML).toNotMatch(/hello/)
+          expect(node.firstChild.innerHTML).not.toBeDefined()
```

Comment:
Yeah, this looks funky. Thanks for noticing it. Will have a look... I need to get back into the code I was doing in April. 🙀

Unchecked this point in my TODO:
https://github.com/skovhus/jest-codemods/pull/39